### ISSUE TITLE
[DataCap Application] <SFF Organization> - <SYDNEY FILM FESTIVAL>

### ISSUE THREAD

**Issue by SFForganization** [OpenGate01/Allocator-Pathway-OpenGate01#18]
### Version

1

### DataCap Applicant

SFF Organization

### Project ID

06

### Data Owner Name

SFF Organization

### Data Owner Country/Region

Australia

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.sff.org.au/

### Social Media Handle

https://x.com/sydfilmfest/

### Social Media Type

Twitter

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

6PiB

### Expected size of single dataset (one copy)

600TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

1.5PiB

### On-chain address for first allocation

f1qiooeuhslmz5hh5fxapa7jgkuk55o3zkywwivga

### Data Type of Application

Public, Open Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Sydney Film Festival takes place at its flagship venue the State Theatre, as well as cinemas in the CBD, Newtown, Cremorne, Randwick, and more, screening films you won’t usually find in the multiplex. 12 films are selected for the Official Competition, which celebrates “courageous, audacious and cutting-edge” cinema with a $60,000 cash prize. Sydney Film Festival also presents eight other cash awards over the course of the festival. 

Sydney Film Festival is supported by the NSW Government through Screen NSW; the Federal Government through Screen Australia; and City of Sydney the Festival’s Principal Local Government Partner. Sydney Film Festival also runs the Travelling Film Festival year-round, taking the best of international and Australian cinema to 22 regional centres around the country.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Event Management and Operations Data:

Film schedules and event details
Venue information and logistics
Ticketing systems and booking records
Volunteer management data
Film and Media Content:

Digital copies or previews of films (for distribution or screening)
Trailers, promotional videos, and advertisements
Media archives from previous festivals (photos, videos, interviews)
Customer Data:

Audience profiles (contact details, demographics, preferences)
Ticket purchase history and participation data
Email subscription lists and marketing preferences

Survey and Feedback Data:

Audience surveys and feedback after screenings or events
Analysis of attendee experiences to improve future festivals
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Australia

### Please share a sample of the data

```text
https://drive.google.com/drive/folders/1oh3aMdQfmpF5B12JMOLP5eriLbGRn7tk?usp=sharing
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Weekly

### For how long do you plan to keep this dataset stored on Filecoin

Permanently

### In which geographies do you plan on making storage deals

Greater China, Africa, North America, Europe, Australia (continent)

### How will you be distributing your data to storage providers

HTTP or FTP server, IPFS

### How did you find your storage providers

Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
will post it in the comment when sps are confirmed
```

### How do you plan to make deals to your storage providers

Boost client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by OpenGate01** [comment]
Thank you for your recognition of Filecoin. We would like to schedule a time for an on-site visit, if possible.

**Comment by SFForganization** [comment]
Yes.Can we have an online docking on September 24, 10:00 am (UTC)? Or what time is convenient for your calendar?

**Comment by OpenGate01** [comment]
This time works for us. Please prepare more data samples and the company's registration documents.Thank you.

**Comment by SFForganization** [comment]
> This time works for us. Please prepare more data samples and the company's registration documents.Thank you.

Hi!We created a google meeting room,pls join here 3 hours later :
Datacap application
Tuesday, September 24 · 7:00 – 8:00pm
Time zone: America/Los_Angeles
Google Meet joining info
Video call link: https://meet.google.com/bte-zzgo-nws

We will present more data sample in the online session,and send you the company's registration documents.
Looking forward to talk with you!Thanks!

**Comment by OpenGate01** [comment]
It was a very successful meeting, and we hope to sponsor your event next year. Using Filecoin's storage method is highly suitable for movies. I believe a film doesn't actually need that many backups; 5-6 copies should be sufficient.

**Comment by SFForganization** [comment]
Certainly, we anticipate having more collaborations with you as we are both located in Sydney. Moreover, 5-6 copies are fine for us. What else do we need to do?

**Comment by OpenGate01** [comment]
I'm sorry, but we are out of datacap.Here’s a good news: we are developing a download tool that eliminates the need to use Lotus command line for downloading movie files. This makes the product experience much closer to Web2.

**Comment by SFForganization** [comment]
> I'm sorry, but we are out of datacap.Here’s a good news: we are developing a download tool that eliminates the need to use Lotus command line for downloading movie files. This makes the product experience much closer to Web2.

Sounds great,could you send me more details about this tool?
And when can we applply for the datacap again?

**Comment by OpenGate01** [comment]
We will notify you once we have the datacap.

**Comment by SFForganization** [comment]
> We will notify you once we have the datacap.

Hi, any updates regarding the quota approval?

If you need any information about our organization, feel free to reply here or send us an email.

Thank you!

**Comment by OpenGate01** [comment]
I'm very sorry, but we don't have the DC quota at the moment. It should be approved soon.Thank you.

**Comment by SFForganization** [comment]
> I'm very sorry, but we don't have the DC quota at the moment. It should be approved soon.Thank you.

Thank you for the update. I understand the situation and appreciate your efforts to get the DC quota approved. Please let me know once it's available, and I look forward to proceeding when the approval comes through.

**Comment by OpenGate01** [comment]
Sorry, we are waiting for the DC quota.
